build.sbt

```diff
@@ -90,9 +90,8 @@ val baseSettings = Seq(
   version := buildVersion,
   ThisBuild / scalaVersion := scala213,
   crossScalaVersions := crossVersionAll,
-  crossVersion := CrossVersion.binary,
```

Comment:
That's why the cross dependency.
I published the artifacts locally and their dependencies looks fine. I'll work more on the js tests, but the publishing of the jvm part should be fine again.